### ISSUE TITLE
License key

### ISSUE THREAD

**Issue by mmillican** [dave/dbcombo#1]
You were surprised to know there was an active user in 2017; how about 2020? :)

Anyways, my employer is still using this on an old application and we need to move it to a new server. It looks like the license is restricted to a specific machine (the old server) but we can't tell for sure. Is there any way you can provide a different license key still?

**Comment by dave** [comment]
Oh my lord.

**Comment by dave** [comment]
Hahaha ok let me see...

**Comment by dave** [comment]
Did you try the key that's in this file?

https://github.com/dave/dbcombo/blob/master/WebSite1/Default.aspx

I'm sure I had an unlimited code somewhere... that might be it? But I don't have my emails from 20 years ago! haha

**Comment by dave** [comment]
With the source code it should be possible to reconstruct a working code...

**Comment by dave** [comment]
(or just compile a version that doesn't check the registraion key)

**Comment by dave** [comment]
It should be possible to reverse engineer a working key from this function:

https://github.com/dave/dbcombo/blob/master/Cambro.Web.DbCombo/DbCombo.cs#L5373-L5450

... I haven't written a line of c# for like 15 years though, so I won't be much help 😉

**Comment by dave** [comment]
BTW if you manage to create a working key, let me know and I'll add it to the readme. Someone else will probably come along in 5 years time 🤣

**Comment by mmillican** [comment]
Haha, I know crazy, right? I tried one key, but don't remember which one it was. I'll try the one you linked to and see what happens.

**Comment by mmillican** [comment]
Sadly the key in the demo project is still giving me 

```
DbCombo error: You have entered an invalid registration key. Please see www.dbcombo.net for a valid key." Not sure if it's a version difference or what.
```

We tried compiling a new version without the license check last week, but ran into a whole bunch of issues with .NET versions and references when it was deployed. Figured I may as well reach out and see if you had anything around still.

**Comment by mmillican** [comment]
I realize this is a long shot, but do you by chance have the code around for `UrlTextSerialize` still?

**Comment by dave** [comment]
Hmmm I think there was another project that generated the codes, but I don't think I have the source anywhere...

**Comment by dave** [comment]
Yeah I think your best bet it to recompile it with the registration check removed. You might [try this version](https://www.dropbox.com/s/eopp6nnwoyfw5li/DbCombo.zip?dl=1)... Not sure if that's the version you have but in my archives it's in a dir called `DbCombo - for .net 3.5`, so it might be more recent?

**Comment by mmillican** [comment]
Looks like there's some framework version mismatches (with our server) in there unfortunately. Which is similar to the issue we were having when we tried to compile the source without checking the key.

**Comment by mmillican** [comment]
Thanks for your help looking into this!

**Comment by dave** [comment]
Aaah sorry I could be more help!
